validator/accounts/wallet_create.go

```diff
@@ -22,6 +23,14 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	// #nosec G101 -- Not sensitive data
+	newMnemonicPassphraseYesNoText = "(Advanced) Do you want to setup a '25th word' passphrase for your mnemonic? [y/n]"
```

Comment:
should this be here with wallet_recover.go changed

Reply:
similar to the inputNumAccounts function, this const is used in both places (line 179 of this file). once I migrate wallet_create.go to the other package, there will just be the one const shared within the package!